src/lib/react/VideoView.tsx

```diff
@@ -57,5 +58,13 @@ export default ({ muted, stream, onResize, ...rest }: VideoViewProps) => {
         }
     }, [muted, stream, videoEl]);
 
-    return <video ref={videoEl} autoPlay playsInline {...rest} />;
+    return (
+        <video
+            ref={videoEl}
+            autoPlay
+            playsInline
+            style={{ transform: mirror ? "scaleX(-1)" : "none", ...rest.style }}
+            {...rest}
```

Comment:
I assume this `{...rest}` is the reason why it does not work for me when `style` prop is provided 🤔 

It probably overwrites the style prop above when I have a `style={{width:"200px"}}` prop. 😢 

Either the `style` prop should be omitted from rest; or should be extracted earlier; or the `{...rest}` should go above the `style={{mirroring condition}}` prop here.

Reply:
actually I just moved the `{...rest}` above the style. It was easier. Should work now 👍